env.sample

```diff
@@ -1,40 +1,56 @@
 PROJECT_NAME=dead_songs
+DEBUG=True
 
 # the database superuser name - this is the default
-POSTGRES_USER=postgres
+DEVELOPMENT_POSTGRES_USER=postgres
 
 # the database name the API will connect to - "dbname" in most PostgreSQL command-line tools
-POSTGRES_NAME=dead_songs
+DEVELOPMENT_POSTGRES_NAME=dead_songs
 
-# the database owner - automatic restore needs this (most likely only used in dev)
-DATABASE_OWNER=sampleuser
+# the database owner - automatic restore needs this
+DEVELOPMENT_DATABASE_OWNER=sampleuser
 
 # *service* name (*not* image name) of the database in the Docker network
-POSTGRES_HOST=db_development
+DEVELOPMENT_POSTGRES_HOST=db_development
 
 # port the database is listening on in the Docker network
-POSTGRES_PORT=5432
+DEVELOPMENT_POSTGRES_PORT=5432
 
 # password for the PostgreSQL database superuser in the database container
-POSTGRES_PASSWORD=sit-down-c0mic
+DEVELOPMENT_POSTGRES_PASSWORD=sit-down-c0mic
 
 # the password for the teams
-TEAM_PASSWORD=d0wn!0ff!a!duck
+DEVELOPMENT_TEAM_PASSWORD=d0wn!0ff!a!duck
 
 # Django secret key in the API container
-DJANGO_SECRET_KEY=r0ck.ar0und.the.c10ck
+DEVELOPMENT_DJANGO_SECRET_KEY=r0ck.ar0und.the.c10ck
+
+# the database superuser name - this is the default
+PRODUCTION_POSTGRES_USER=postgres
+
+# the database name the API will connect to - "dbname" in most PostgreSQL command-line tools
+PRODUCTION_POSTGRES_NAME=dead_songs
+
+# *service* name (*not* image name) of the database in the Docker network
+PRODUCTION_POSTGRES_HOST=aws_ip
+
+# port the database is listening on in the Docker network
+PRODUCTION_POSTGRES_PORT=5432
+
+# password for the PostgreSQL database superuser in the database container
+PRODUCTION_POSTGRES_PASSWORD=sit-down-c0mic
+
+# Django secret key in the API container
+PRODUCTION_DJANGO_SECRET_KEY=r0ck.ar0und.the.c10ck
 
 # Docker container registry host
 DOCKER_REPO=
 
 # subfolder/namespace in registry
 DOCKER_REPO_NAMEPSACE=
 
-# (Production) Docker image name
-DOCKER_IMAGE=backend-examplar-service
-
-# (Production) Docker service name
-DOCKER_SERVICE=backend-examplar-service
```

Comment:
We use the naming convention _team_-service for DOCKER_IMAGE and DOCKER_SERVICE to differentiate all the backend service docker images in production, so leave this the backend-exemplar way.